lib/frozen_record/test_helper.rb

```diff
@@ -0,0 +1,41 @@
+module FrozenRecord
+  module TestHelper
+    NoFixturesLoaded = Class.new(StandardError)
+
+    def self.load_fixture(model_class, alternate_base_path)
+      @cache ||= {}
+
+      ensure_model_class_is_frozenrecord(model_class)
+
+      @cache[model_class] = {
```

Comment:
There's another issue here. If you load the fixtures twice for the same class, the original values will be definitely erased. I think you should use `||=` here.

Reply:
👍 And I think we could also just return early if `@cache.key?(model_class)`, and avoid redundantly setting `auto_reloading` and `base_path`.